ansible/cleanup_runner_disk.yml

```diff
@@ -0,0 +1,33 @@
+---
+- hosts: cleanup_runners
+  remote_user: root
+  name: cleanup runners disk
+  gather_facts: false
+
+  vars:
+    box_initial_name: "freeipa-VAGRANTSLASH"
+    default_provider: "libvirt"
+
+  vars_prompt:
+    - name: box_name
+      prompt: "Name of the box to be removed (eg: ci-master-f27)"
+      private: false
+
+    - name: box_version
+      prompt: "Version of the box to be removed (eg: 0.1.0)"
+      private: false
+
+    - name: user_confirmation
+      prompt: "Are you sure you want to remove this box? Answer with YES or NO"
+      default: "NO"
+      private: false
+
+  tasks:
+    - name: remove vagrant box
+      when: user_confirmation == "YES" and box_name != "" and box_version != ""
+      shell: vagrant box remove freeipa/{{ box_name }} --provider {{ default_provider }} --box-version {{ box_version }} 
+
+    - name: remove libvirt image
+      when: user_confirmation == "YES" and box_name != "" and box_version != ""
```

Comment:
This would be a nice place for a [`block`](http://docs.ansible.com/ansible/latest/playbooks_blocks.html) to avoid copy-pasting the condition.